src/library/fixitem/FixItemControl.java

```diff
@@ -0,0 +1,72 @@
+package library.fixitem;
+import library.entities.Item;
+import library.entities.Library;
+
+public class FixItemControl {
+    
+    private enum ControlState { INITIALISED, READY, INSPECTING };
+    private ControlState state;
+    private FixItemUI ui;
+    
+    private Library library;
+    private Item currentItem;
+
+
+    public FixItemControl() {
+        this.library = Library.getInstance();
+        state = ControlState.INITIALISED;
+    }
+    
+    
+    public void setUI(FixItemUI ui) {
+        if (!state.equals(ControlState.INITIALISED))
+            throw new RuntimeException("FixItemControl: cannot call setUI except in INITIALISED state");
```

Comment:
PR 46 D1 CSG#7 Curly brackets must be in use.

Reply:
[37e8a84](https://github.com/elliotmitchum/ITC205/pull/46/commits/37e8a84c403454296d1bdb09cc9ca7b54552cde0)